javascript/sdk/test/templates/template.go

```diff
@@ -2,7 +2,7 @@
 ID: $ID
 NAME: $NAME
 UNIT: $UNIT
-CREATED: $TIME
+CREATED: $CREATED
```

Comment:
you know you're diverging from the python tmeplate? https://github.com/preludeorg/libraries/blob/master/python/cli/prelude_cli/templates/template.go